axelrod/result_set.py

```diff
@@ -358,6 +358,33 @@ def _build_normalised_state_distribution(self):
             norm.append(counters)
         return norm
 
+    @update_progress_bar
+    def _build_normalised_state_to_action_distribution(self):
+        """
+        Returns
+        ----------
+
+            Normalised state distribution. A list of lists of counter objects:
+
+            Dictionary where the keys are the states and the values are a
+            normalized counts of the number of times that state goes to a given
+            action.
+        """
+        norm = []
+        for player in self.state_to_action_distribution:
+            counters = []
+            for counter in player:
+                norm_counter = Counter()
+                for state in [(C, C), (C, D), (D, C), (D, D)]:
+                    total = counter[(state, C)] + counter[(state, D)]
+                    if total > 0:
```

Comment:
`if total == 0` then we add nothing to the list? just checking that is the intention rather than None or an empty dict.